app/views/items/_item.html.erb

```diff
@@ -0,0 +1,9 @@
+<%= turbo_frame_tag item do %>
+  <%= button_to "#{item.label} => #{item.votes.count}",
+        votes_path(
+          user_id: 1,
+          item_id: item.id
+        ),
+        method: :post, data: { turbo_frame: '_top' } %>
```

Comment:
I read that 

> There is a special frame called _top that represents the whole page.

> Every page has the "_top" frame by default

 so we can safely remove `data: { turbo_frame: '_top' }` from here because the button is already its own whole section